### PR TITLE
[IMP] account, purchase, sale, l10n_br_sales, l10n_{ar,cl}: Ui fix

### PR DESCRIPTION
**Pull request by malb-odoo**
In this pr: https://github.com/odoo/enterprise/pull/30853 we've merged two
settings into one to make Tax calculation & display more coherent. In the case
of a "Tax per line" computation, the column "Tax Included" is optional=hide in
the backend, but we've elected to put it in the PDF all the time.

This task we will only change the pdf and portal (preview):
1. Remove the "Tax Incl" column on all relevant templates (pdf/portal).
2. Rename the "Tax Excl" column to "Amount".

task-3552656

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/sale/report/ir_actions_report_templates.xml
Comment (Feyensv) [inline addons/sale/report/ir_actions_report_templates.xml:109]: What is this change doing here ? :thinking:

Comment (malb-odoo) [reply addons/sale/report/ir_actions_report_templates.xml:109]: ooops should not be there, mistake on my part :eyes:

**Comment by malb-odoo** [comment]
waiting on po approval, better to let it in draft

**Review by FlorianGilbert — APPROVED** [review]
Hello, the PR looks good to me, but I'm not convinced by the commit message, shouldn't we change it for [IMP] (according to the task) or [FIX]? We do not really revert a commit, we just changed our mind about how we want to display these values.

**Review by FlorianGilbert — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**
